device-types/Aruba/Aruba7005.yaml

```diff
@@ -0,0 +1,19 @@
+manufacturer: Aruba
+model: Aruba7005
+slug: aruba7005
+is_full_depth: no
```

Comment:
missing power ?

Reply:
Honestly, I couldn't figure out what to put for power since I think its an external power supply so most likely a DC barrel connector. I've never even seen a 7005, we only have one in an office thats on a different continent than me.

Reply:
There is port 3 (or 0) can accept PoE input... 

and for external power, it is from 7005 Ordering guide... _AP-AC-12V30A 12V/30W AC/DC Desktop Style 1.7/4.0/9.5mm Circular 90 Deg Plug DoE Level VI Adapter_

Reply:
You'll want to create another pull request to add the power info for the 7005.